pseudocode.md

```diff
@@ -1,6 +1,14 @@
 # Hangman (Pseudocode)
 
 ```
-Write pseudocode here, within the tripple backticks.
-
+1. P1 Chooses a word or series of words
+2. Break up word into seperate letters
+3. Start loop
+4. P2 Chooses letter or word
+5. if P1 chooses a letter and the letter is in the word, reveal all spots where it is found
+6. elif player chooses a letter and letter is not in the word, add a body part to the hangman
+7. elif the player chooses the wrong word, add a body part
```

Comment:
With the way we went forward with the class version, think of a way we can implement adding a body part with a whole-word guess rather than simply automatically losing.